src/components/ChatMap/ChatMap.jsx

```diff
@@ -1,33 +1,19 @@
 import { useState } from 'react';
-import { useDispatch, useSelector } from 'react-redux';
 import PropTypes from 'prop-types';
 import { GeoJSON } from 'react-leaflet';
 import mapData from '@/data/countries.json';
 import 'leaflet/dist/leaflet.css';
 import '@/css/ChatMap.css';
 
-import { sendDataCountryToBackend } from '@/redux-store/AuthOperations/AuthOperations';
-import { getUser } from '@/redux-store/selectors';
 import {
   CountryName,
   ShowCountry,
   MainMapBlock,
   MapWrapper,
 } from './ChatMapStyled';
-// TODO
-// import { useMap } from 'react-leaflet/hooks';
 
 const ChatMap = ({ closeMap }) => {
-  const dispatch = useDispatch();
-  const userId = useSelector(getUser)?.id;
   const [selectedCountry, setSelectedCountry] = useState(null);
-  const [countryData, setCountryData] = useState({});
-
-  // TODO
-  // const onCountryHover = e => {
-  //   const countryName = e.target.feature.properties.ADMIN;
-  //   setSelectedCountry(countryName);
-  // };
 
   const color = [
```

Comment:
This variable and the one below should be moved out of the component.

Reply:
Done